upgrade/Stage_0_Prerequisites.md

```diff
@@ -412,17 +413,57 @@ example showing how to find the IUF activity.
    disables the components and does not clear their configuration states or error counts. When the
    nodes are rebooted to their new images later in the CSM upgrade, they will automatically be
    enabled in CFS, and node personalization will occur.
+  
+    1. `(ncn-m001)` Update CFS on worker nodes and master nodes.
 
-   ```bash
-   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
-       --no-config-change --config-name "${CFS_CONFIG_NAME}" --no-enable --no-clear-err
-   ```
+        Get master nodes' and worker nodes' xnames.
 
-   Successful output will end with the following:
+        ```bash
+        XNAME_WORKERS=$(cray hsm state components list --role Management --subrole Worker --type Node --format json |
+          jq -r '.Components | map(.ID) | join(",")')
+        XNAME_MASTERS=$(cray hsm state components list --role Management --subrole Master --type Node --format json |
+          jq -r '.Components | map(.ID) | join(",")')
+        echo "${XNAME_MASTERS},${XNAME_WORKERS}"
```

Comment:
super nitpicky, but `WORKER_XNAMES` and `MASTER_XNAMES` more accurately describes what the variable holds.

```suggestion
        WORKER_XNAMES=$(cray hsm state components list --role Management --subrole Worker --type Node --format json |
          jq -r '.Components | map(.ID) | join(",")')
        MASTER_XNAMES=$(cray hsm state components list --role Management --subrole Master --type Node --format json |
          jq -r '.Components | map(.ID) | join(",")')
        echo "${MASTER_XNAMES},${WORKER_XNAMES}"
```